images/build/Dockerfiles/Dockerfile

```diff
@@ -79,24 +79,24 @@ RUN set -ex \
  # 1.1 sdk <-- 1.0 runtime's sdk
  && mkdir $NET_CORE_APP_10 \
  && ln -s $NET_CORE_APP_10 1.0 \
- && ln -s $sdksDir/$DOT_NET_CORE_11_SDK_VERSION $NET_CORE_APP_10/sdk \
```

Comment:
FYI:
.net core has runtime and sdk versions. AppService's version list is based on the runtime version, so we try to resolve the actual sdk version by reading this metadata file `sdkVersion.txt`
This file is created statically like this here in Dockerfile as well as when generating script to dynamically install the sdk